pages/index.js

```diff
@@ -57,75 +64,95 @@ export default function HomePage() {
-          </StyledSection>
-          
-          <PetForm selectedPet={selectedPet} onSubmit={handleSubmit} />
+            <StyledHeading $variant="select">
+              Select a Survival Pet
+            </StyledHeading>
+            <StyledSection>
+              <PetList
+                onSelectPet={handleSelectPet}
+                selectedPet={selectedPet}
+              />
+            </StyledSection>
+            <PetForm selectedPet={selectedPet} onSubmit={handleSubmit} />
           </StyledGrid>
         </>
       )}
 
       {mode === "livingroom" && (
         <>
           <StyledGrid>
-          <StyledHeading $variant="livingroom">Living Room</StyledHeading>
-          {!isDead ? (
-            <StyledSection>
-              <CurrentPet selectedPet={selectedPet} />
-            </StyledSection>
-          ) : (
-            <StyledSection>
-              <TombstoneButton
-                selectedPet={selectedPet}
-                onMode={handleMode}
-                onDeletePet={handleDeletePet}
-              />
-            </StyledSection>
-          )}
-          <CurrentPetStats
-            isDead={isDead}
-            selectedPet={selectedPet}
-            onMode={handleMode}
-          />
+            <StyledHeading $variant="livingroom">Living Room</StyledHeading>
+            {!isDead ? (
+              <StyledTest>
+                <div></div>
+                <StyledSection>
+                  <CurrentPet selectedPet={selectedPet} />
+                </StyledSection>
+                <InteractionMenu onIncreaseStats={handleIncreaseStats} />
+              </StyledTest>
+            ) : (
+              <StyledSection>
+                <TombstoneButton
+                  selectedPet={selectedPet}
+                  onMode={handleMode}
+                  onDeletePet={handleDeletePet}
+                />
+              </StyledSection>
+            )}
+            <CurrentPetStats
+              isDead={isDead}
+              selectedPet={selectedPet}
+              onMode={handleMode}
+            />
           </StyledGrid>
         </>
       )}
 
       {mode === "edit" && (
         <>
           <StyledGrid>
-          <StyledHeading $variant="livingroom">Living Room</StyledHeading>
-          <StyledSection>
-            <CurrentPet selectedPet={selectedPet} />
-          </StyledSection>
-          <EditForm
-            selectedPet={selectedPet}
-            onSubmit={handleSubmit}
-            onMode={handleMode}
-          />
+            <StyledHeading $variant="livingroom">Living Room</StyledHeading>
+            <StyledSection>
+              <CurrentPet selectedPet={selectedPet} />
+            </StyledSection>
+            <EditForm
+              selectedPet={selectedPet}
+              onSubmit={handleSubmit}
+              onMode={handleMode}
+            />
           </StyledGrid>
         </>
       )}
 
       {mode === "eliminate" && (
```

Comment:
Just one question out of interest - what is the difference between "eleminate" pet and "delete" pet?